[*] Has Many Relationships - Fix sorting on hasMany fields in related data (#360)
ForestAdmin/forest-express-sequelize: CHANGELOG.md

# Change Log

## [Unreleased]

> ### Fixed

Reviewer: I changed my mind (Sorry for the back and forth) but maybe we should define this change as a feature for a more positive vision.
```suggestion
### Added
```

> - Data summary/RelatedData - Fix sorting by referenced field.

Reviewer: This fix description is too generic in my opinion.
You should detail in which case the sorting on belongsTo relationships was broken.

Reviewer: I would start the description with:
```suggestion
- Has Many Relationships - ...
```

> - Has Many Relationships - Fix sorting on hasMany fields in related data.

Reviewer: ```suggestion
- Has Many Relationships - Fix sorting on belongsTo relationship columns in related data.
```

> - Has Many Relationships - Fix sorting on belongsTo relationship columns in related data.

Reviewer: ```suggestion
- Has Many Relationships - Enable sorting on belongsTo relationship columns in related data.
```
